misc/helm-charts/operator/README.md.gotmpl

```diff
@@ -202,6 +202,106 @@ kubectl logs -l app.kubernetes.io/name=materialize-operator -n materialize
 
 For more detailed information on using and troubleshooting the Materialize operator, refer to the [Materialize documentation](https://materialize.com/docs).
 
+## Upgrading Materialize
+
+Once you have the Materialize operator installed, with the operator managing your Materialize environments, you can upgrade the operator and the environments independently.
```

Comment:
if they're independent, is there a risk the Helm chart gets bumped too far such that a `Materialize` instance can't upgrade properly?